tests/integration/test_pg_connector.py

```diff
@@ -8,42 +8,45 @@
 pytestmark = pytest.mark.asyncio
 
 
-async def test_get_connection(connection):
-    dsn = connection.dsn
-    async with await aiopg_connector.get_connection(dsn=dsn) as new_connection:
+@pytest.fixture
```

Comment:
rename file as test_aiopg_connector

Reply:
do you plan to do it with a separate PR?

Reply:
Damn, you're right